### PR TITLE
[ft-158486263] add friend page

### PR DESCRIPTION
**Pull request by Ogollah**
#### What does this PR do?
Delete friend page 
#### Reason 
Overlooked project requirements

### CODE REVIEW

**Comment by meshack-mbuvi** [comment]
The design looks good.A curious though, what am seeing detail of rejected/accepted rides. Was thinking I should be seeing just the names and maybe a few words about my friends. How do you see it?

**Comment by Ogollah** [comment]
@meshack-mbuvi That is a good idea, on my side, I was looking at the list of friends with ride request so that I can comment on their ride status. How does that sound?

**Comment by meshack-mbuvi** [comment]
My opinion is; you should just have a friend page listing one's friends. You can have name and few words about the friend. I don't thin you should show the status of ride request.Maybe that could be personal to a given user.

File: UI/templates/friend.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        </div>
+    </div>
+
+    <!--footer fixed-->
+    <div class="footer">
+        <p>Ride-my-way &copy; 2018</p>
+    </div>
+</body>
+
+</html>
```
Comment (LWanjiru) [inline UI/templates/friend.html]: add a new line here and at the end of every file

File: UI/templates/welcome.html [hunk 2]
```diff
@@ -21,16 +25,18 @@
...
     </div>
     <!--footer fixed-->
     <div class="footer">
         <p>Ride-my-way &copy; 2018</p>
     </div>
-    
+
 </body>
+
 </html>
```
Comment (LWanjiru) [inline UI/templates/welcome.html]: see comment above

**Review by LWanjiru — CHANGES_REQUESTED** [review]
@Ogollah I am a little concerned about this task `A user should be able to see a list of friends`, please colour on where this falls in the project requirements. What inspired this page?

File: UI/templates/welcome.html (hunk 2 above)
Comment (Ogollah) [reply UI/templates/welcome.html]: I have noticed where I went wrong. I overlooked the optional feature "Users can only see and respond to ride offers from their friends on the application", Thought it was nice for friends to comment on their friends ride status. That is why I introduced friends.

**Review by LWanjiru — CHANGES_REQUESTED** [review]
I am curious as to why you deleted the screenshot? What is this PR about? Also, why did you prefer to delete the screenshot rather than answer the question I posed?

**Comment by Ogollah** [comment]
@LWanjiru I consulted the project guideline and found out that I overlooked the requirement at that point. The requirements are if a Users can only see and respond to ride offers from their friends on the application. My idea was for a user to review a friends request status.

**Pull request merged**